src/frontend/next/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "next dev -p 8000",
-    "build": "next build",
+    "build": "next build && next export",
```

Comment:
Do we need to modify the build command for the frontend? I think this might break something, or at least delay deployment times due to the export. Let me know!